contracts/src/main/kotlin/jp/datachain/corda/ibc/ics2/ClientType.kt

```diff
@@ -3,10 +3,18 @@ package jp.datachain.corda.ibc.ics2
 import net.corda.core.serialization.CordaSerializable
 
 @CordaSerializable
-enum class ClientType {
-    CordaClient,
-    FabricClient,
-    SoloMachineClient,
-    TendermintClient,
-    LoopbackClient,
+enum class ClientType(private val string: String) {
```

Comment:
@siburu 
I'm not sure if this should be included in this PR, but we need to comply with the spec for Client identifier in ICS-024.
ref. https://github.com/cosmos/ibc/tree/master/spec/core/ics-024-host-requirements#paths-identifiers-separators
https://github.com/hyperledger-labs/yui-ibc-solidity/pull/37

In fabric, currently uses `hyperledgerfabric` as Client identifier. (but I think we should use `hyperledger-fabric` instead of it to follow the format in ibc-solidity)

Reply:
Thank you for your advice!
This will be a problem during E2E testing anyway.
Therefore I'm replacing this with `FabricClient("hyperledgerfabric")` now.

Reply:
Ah I found the following in ICS-24.

Port identifier | Client identifier | Connection identifier | Channel identifier
-- | -- | -- | --
2 - 64 | 9 - 64 | 10 - 64 | 10 - 64

I'll also replace `CordaClient("corda")` with `CordaClient("corda-ibc")`.

Reply:
Done.